src/openrct2/sprites.h

```diff
@@ -840,6 +840,14 @@ enum {
     
     SPR_G2_EYEDROPPER = SPR_G2_BEGIN + 123,
 
+    SPR_G2_CHAR_BEGIN = SPR_G2_BEGIN + 124,
+    SPR_G2_AE_UPPER = SPR_G2_CHAR_BEGIN,
+    SPR_G2_AE_LOWER = SPR_G2_CHAR_BEGIN + 1,
+    SPR_G2_O_STROKE_UPPER = SPR_G2_CHAR_BEGIN + 2,
+    SPR_G2_O_STROKE_LOWER = SPR_G2_CHAR_BEGIN + 3,
+
+    SPR_G2_GLYPH_COUNT = (SPR_G2_O_STROKE_LOWER - SPR_G2_CHAR_BEGIN) + 1,
```

Comment:
This is off, I think, because you have only taken the small font into account.

Reply:
This is how the vanilla glyph count works as well. It's more useful like this anyway.